src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/email/ReportSenderTest.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.services.email;
+
+import com.icegreen.greenmail.junit.GreenMailRule;
+import com.icegreen.greenmail.util.ServerSetupTest;
+import org.apache.commons.mail.util.MimeMessageParser;
+import org.assertj.core.util.Arrays;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import uk.gov.hmcts.reform.bulkscanprocessor.services.reports.ReportsService;
+
+import java.util.Properties;
+import javax.mail.Address;
+
+import static java.time.LocalDate.now;
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ReportSenderTest {
```

Comment:
Would be nice to add tests to cover `MessagingException` and `IOException`

Reply:
added extra test